### PR TITLE
Fix some tests failing in `--pass check` mode

### PR DESCRIPTION
**Pull request by petrochenkov**
r? @RalfJung

### CODE REVIEW

File: src/test/ui/lint/lint-exceeding-bitshifts.rs [hunk 1]
```diff
@@ -2,8 +2,8 @@
 //[noopt]compile-flags: -C opt-level=0
 //[opt]compile-flags: -O
 //[opt_with_overflow_checks]compile-flags: -C overflow-checks=on -O
-
 // build-pass
+// ignore-pass (emit codegen-time warnings and verify that they are indeed warnings and not errors)
```
Comment (RalfJung) [inline src/test/ui/lint/lint-exceeding-bitshifts.rs:6]: Did you mean "emits" and "verifies"? I cannot entirely parse this comment.

Comment (petrochenkov) [reply src/test/ui/lint/lint-exceeding-bitshifts.rs:6]: I copypasted it from other tests.
"[Let the test] emit codegen-time warnings and verify that they are indeed warnings and not errors.", I guess.

Comment (RalfJung) [reply src/test/ui/lint/lint-exceeding-bitshifts.rs:6]: Okay. I'd still prefer if you could clarify here, I found it unclear if this was asking the reader to do something or the compiler or what.

Comment (petrochenkov) [reply src/test/ui/lint/lint-exceeding-bitshifts.rs]: Updated.

**Comment by RalfJung** [comment]
Thanks!
@bors r+ rollup

**Pull request merged**